tssilomon: use #!/bin/sh as interpreter

joufellasfl opened the pull request:
Switch to /bin/sh as the script content is POSIX compliant.

This avoids pulling a nonessential dependency (many systems solely rely on Busybox ash), and facilitates integration into systems where GPLv3 licensed components are a challenge.

ts-kris commented:
Thank you for pointing this out. I'll queue up some testing get this merged, and pushed out to build systems.

joufellasfl committed:
tssilomon: use #!/bin/sh as interpreter

Switch to /bin/sh as the script content is fully POSIX compliant. This
avoids pulling a nonessential dependency (many systems rely solely on
Busybox ash), and facilitates integration into systems where GPLv3
licensed components like Bash is a challenge.

joufellasfl commented:
@ts-kris I sent an update on the commit header and message, please use the latest version if you merge it.
